examples/amd-azure/README.md

```diff
@@ -39,58 +42,59 @@ The throughput value reported comes from measuring the number of finished infere
+For this benchmarking script, users must specify the topology of their system with a JSON file. This file includes a list of lists of cores. 
+
+One list of cores will contain the processor IDs that one of the worker processes will run on, and should reflect the topology of the system. For performance, one list of cores in the JSON topology file should contain the list of cores that are on the same socket, are on the same NUMA node, or share the same L3 cache. 
+
 The `/examples/amd-azure` directory contains two example JSON files that can be used:
-    * `azure_hb120_v3.json`, which is suitable for use on a Microsoft Azure HB120\_V3 instance. You may notice that not every process will use the same number of cores when using this topology. This is because some of the CCXs on this instance type have some cores dedicated to running the hypervisor.
-    * `amd_epyc_7713.json`, which is suitable for a two socket system with AMD EPYC 7713 processors. This file will also work for a one socket system if the proper parameter for `nstreams` is passed into `multi_process_benchmark.py`.
+
+- `azure_hb120_v3.json`, which is suitable for use on a Microsoft Azure HB120\_V3 instance. You may notice that not every process will use the same number of cores when using this topology. This is because some of the CCXs on this instance type have some cores dedicated to running the hypervisor.
+- `amd_epyc_7713.json`, which is suitable for a two-socket system with AMD EPYC 7713 processors. This file will also work for a one-socket system if the proper parameter for `nstreams` is passed into `multi_process_benchmark.py`.
 
 ## Usage
 
-First install the requirements using
+First, install the requirements using
 ```bash
 pip install -r requirements.txt
 ```
 `multi_process_benchmark.py` uses `py-libnuma` to control the CPU affinity and memory policy of each individual stream to optimize performance.
 
-In most cases, good performance will be found in the default options so it can be as simple as running the command with a SparseZoo model stub or your local ONNX model.
+In most cases, good performance will be found in the default options so it can be as simple as running the command with a [Neural Magic SparseZoo](https://sparsezoo.neuralmagic.com) model stub or your local ONNX model.
 
 ```bash
 python multi_process_benchmark.py <path/to/model> <path/to/topology/file>
 ```
 
-Running with `multi_process_benchmark.py -h` or `--help` provides usage options:
+Executing `multi_process_benchmark.py -h` or `--help` provides the following usage options:
 
 ```
 usage: python multi_process_benchmark.py [-h] [-b BATCH_SIZE] [-nstreams NUM_STREAMS] [-shapes INPUT_SHAPES] [-t TIME] [-w WARMUP_TIME]
                                   [-pin {none,core,numa}] [-q]
                                   model_path topology_file
 
-Benchmark ONNX models in the DeepSparse Engine
```

Comment:
Everything from here down should be consistent with the parse_args function in multi_process_benchmark.py

Reply:
I can apply these edits to that file @tlrmchlsmth

Reply:
Let me know if you need any help

Reply:
Made changes and will ask for a re-review. parse_args for -h and -help must come default from elsewhere and I didn't update the descriptive language there; honestly, not a showstopper.